spec/app_spec.rb

```diff
@@ -1,174 +1,177 @@
 require 'spec_helper'
 
-describe "app" do
-
-  describe "access control" do
+describe 'app' do
+  describe 'access control' do
     let(:user) { create_test_user(42) }
     # all routes (even nonexistent ones) are covered by the api key
     # /heartbeat is the only exception, covered in the heartbeat tests below
-    let(:urls) { {
-        "/" => 404,
-        "/api/v1/users/#{user.id}" => 200,
-        "/api/v1/users/doesnotexist" => 404,
-        "/selftest" => 200
+    let(:urls) do
+      {
+          '/' => 404,
+          "/api/v1/users/#{user.id}" => 200,
```

Comment:
Oops.  I now see that double-quote was needed to get interpolation.

Here's a strong argument for using double-quotes everywhere:
https://www.viget.com/articles/just-use-double-quoted-ruby-strings
FYI: @clintonb @dianakhuang